Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.py

```diff
@@ -529,10 +532,12 @@ def main():  # pragma: no cover
                 if events:
                     demisto.info(f"Sending events to xsiam with latest event time is: {events[-1]['_time']}")
                     send_events_to_xsiam(events, VENDOR, PRODUCT, should_update_health_module=False)
-                set_integration_context({"offset": offset, "hashed_events_from_previous_run": hashed_events_from_previous_run})
+                set_integration_context({"offset": offset,
+                                         "hashed_events_from_previous_run": list(hashed_events_from_previous_run)})
```

Comment:
why store previous run and not this run? `list(hashed_events_from_previous_run)})`

Reply:
Changed it to {"hashed_events_from_previous_run": list(hashed_events_from_current_run)}

I think it makes sense to keep the context key as "hashed_events_from_previous_run" because we'll only load it on the next run.